Classes/GlobalStateExplorers/DatabaseBrowser/FLEXTableContentViewController.m

```diff
@@ -260,5 +279,18 @@ - (void)executeStatementAndShowResult:(NSString *)statement completion:(void (^_
     } showFrom:self];
 }
 
+- (void)reloadTableDataFromDB {
+    if (self.databaseManager == nil) {
```

Comment:
same, was wondering if we can remove this after we ensure it's non-null

Reply:
Without breaking this out into two separate classes (one for immutable tables and one for mutable tables) the best I can do is to wrap it in a property `self.canRefresh` which checks if we have `self.databaseManager` and `self.tableName`